sitemedia/scss/components/_transcription.scss

```diff
@@ -480,6 +480,33 @@
                 color: var(--on-background);
                 @include typography.body-bold;
             }
+            // rotation controls (desktop)
+            button.rotate {
+                display: none;
```

Comment:
How much trouble to display on mobile and fit in with the existing toggle?

Reply:
Easy to do, but were we waiting on further feedback on mobile rotation to do this?

Reply:
Let's make them available on mobile but hold off on disabling touch rotate. Doesn't hurt to have more than one way to do it and then they can test and compare.